server/router/server.go

```diff
@@ -147,6 +147,10 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int, g http.
 	gMux.Handle("/api/meshmodel/validate", h.AuthMiddleware(http.HandlerFunc(h.ValidationHandler), models.NoAuth)).Methods("POST")
 	gMux.Handle("/api/meshmodel/component/generate", h.AuthMiddleware(http.HandlerFunc(h.ComponentGenerationHandler), models.NoAuth)).Methods("POST")
 	gMux.Handle("/api/meshmodel/components", h.AuthMiddleware(http.HandlerFunc(h.GetAllMeshmodelComponents), models.NoAuth)).Methods("GET")
+	gMux.HandleFunc("/api/meshmodel/components/register", h.RegisterMeshmodelComponents).Methods("POST")
+	gMux.HandleFunc("/api/meshmodel/components/types", h.MeshmodelComponentsForTypeHandler).Methods("GET")
+	gMux.HandleFunc("/api/meshmodel/components/{type}", h.GetMeshmodelComponentsByType).Methods("GET")
```

Comment:
Should these have Auth type assigned?

Reply:
Yes. NoAuth for now. Forgot about it. Making the change now.